src/NewEmail/Form/index.jsx

```diff
@@ -0,0 +1,103 @@
+// Note for the next TEAM:1. For function's you have console.log() to see in console what is happening
+// Note for the next TEAM:2. In this Form component you will find The Form and the Buttons
+// Note for the next TEAM:3. You will find comments before Function for what are they and so on
+import {useState} from "react"
+function Form() {
+    const [email, setEmail] = useState("")
+    const [subject, setSubject] = useState("")
+    const [textArea, setTextArea] = useState("")
+    const [send, setSend] = useState(false)
+    const [cancel, setCancel] = useState(false)
+
+    // Function for cancel button ->
+    const handleCancel = () => {
+        setEmail("")
+        setSubject("")
+        setTextArea("")
+        setCancel(true)
+        setSend(false)
+        console.log("Cancel action ON")
+    }
+
+    // Function for Send button ->
+    const handleSend = () => {
+        if (email.trim() && subject.trim() && textArea.trim()) {
+            setSend(true)
+            setCancel(false)
+
+            // Data sending to API
+            fetch("https://email-client-api.dev.io-academy.uk/email", {
+                method: "POST",
+                headers: {
+                    "Content-Type": "application/json"
+                },
+                body: JSON.stringify({
+                    to: email,
+                    subject: subject,
+                    content: textArea
+                })
```

Comment:
You're missing some data in here as per the API documentation:

```
 {
      "name": "Bob Ross",
      "email": "bob.ross@paintings.com",
      "subject": "Example",
      "body": "Exmaple text",
  }
```

Note: The name can just be set to 'Red Pandas' - it's the name of the person sending the email.